CHANGELOG.md

```diff
@@ -3,6 +3,42 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.22](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.21...v4.0.0-alpha.22)
+
+_Mar 9, 2021_
+
+Big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Implement base foundation for editing a cell (#1025) @dtassone.
+  This is the foundation on which the feature will be built. Currently the newly added methods in the apiRef are not yet document and may be subject to change.
+- 🇩🇪 Added deDE locale (#1117) @LarsKumbier
+- 📜 Fix scrollbar related issue (#1146) @dtassone
+- 🐛 Handle commas in cell values when doing CSV export (#1154) @DanailH
+
+### @material-ui/x-grid@v4.0.0-alpha.22 / @material-ui/data-grid@v4.0.0-alpha.22
+
+#### Changes
+
+- [DataGrid] Add deDE locale (#1117) @LarsKumbier
+- [DataGrid] Fix scrollbar on autopageSize (#1146) @dtassone
+- [DataGrid] Handle special chars when doing CSV export (#1154) @DanailH
```

Comment:
So it's easier to spot what's a fix and what's an enhancement? We very often use "Fix".

```suggestion
- [DataGrid] Fix handling of special chars when doing CSV export (#1154) @DanailH
```